exercises/higher_order_array_methods/exercise-4b.js

```diff
@@ -0,0 +1,28 @@
+function findMaxItems(menu, amount) {
+  // Complete this function
+
+}
+
+// ------------------------------------------------------//
+// Don't edit the code below here                        //
+// ------------------------------------------------------//
+
+function sortMenu(menu) {
+  menu
+    .sort((a, b) => b.price - a.price)
+}
```

Comment:
we still need to fix this:

```suggestion
function sortMenu(menu) {
  return menu.sort((a, b) => b.price - a.price)
}
```